private/buf/buffetch/internal/internal.go

```diff
@@ -211,8 +262,8 @@ type ModuleRef interface {
 // NewModuleRef returns a new ModuleRef.
 //
 // The path must be in the form server/owner/repository/branch[:digest].
-func NewModuleRef(path string) (ModuleRef, error) {
-	return newModuleRef("", path)
+func NewModuleRef(format string, path string) (ModuleRef, error) {
```

Comment:
Same comment whole way down

Reply:
See [answer to the question above](https://github.com/bufbuild/buf/pull/2172#discussion_r1224720727)